runtime/makelib/targets.mk.ftl

```diff
@@ -242,8 +240,8 @@ endif
 UMA_LINK_PATH+=-L. -L$(UMA_PATH_TO_ROOT) -L$(UMA_PATH_TO_ROOT)lib/
 UMA_BUILD_LIBRARIES:=$(foreach lib,$(UMA_LIBRARIES),$(if $($(lib)_alllib),$(lib)))
 UMA_LINK_PATH+=$(foreach lib,$(UMA_BUILD_LIBRARIES),-L$($(lib)_path))
-# Remove duplicate paths
-UMA_LINK_PATH:=$(sort $(UMA_LINK_PATH))
+# remove sorting of UMA_LINK_PATH, changes link order
+#UMA_LINK_PATH:=$(sort $(UMA_LINK_PATH))
```

Comment:
This changes linking on all unix platforms.  This change needs to be removed.

Reply:
Used spec file check to separate logic.